ModelBase/src/nodes/Reference.cpp

```diff
@@ -85,7 +85,8 @@ bool Reference::resolveHelper(bool indirect)
 
 	auto newTarget = computeTarget();
 
-	Q_ASSERT(!newTarget || (newTarget->definesSymbol() && newTarget->symbolName() == name_));
+	//TODO@cyril What is this for?
+//	Q_ASSERT(!newTarget || (newTarget->definesSymbol() && newTarget->symbolName() == name_));
```

Comment:
Actually now that I think about it, it might be possible to declare an attribute directly of type Node*. If that's possible then simply do that and avoid having to deal with ids.